CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Unreleased
 
+- Breaking change: Remove Google Analytics code (migrated to static)
+
+  This code has been [migrated to static](https://github.com/alphagov/static/pull/1795) for use on GOV.UK and is no longer supported. It is being removed from the toolkit because it is now duplicate code and is overriding any changes made in static.
```

Comment:
I wonder if we could word this so it doesnt seem like this code is no longer supported today.

"As this library has been deprecated the code has been migrated to static for use in GOV.UK Publishing"

Reply:
I like it, changed, thanks.